R/dataset_utilities.R

```diff
@@ -220,7 +220,8 @@
     i = paste(
       "The outcome must be NULL,",
       "a factor,",
-      "or a data.frame with a single factor column."
+      "a numeric vector,",
+      "or a data.frame with a single compatible column."
```

Comment:
Ha, I love that I totally had this in mind when I wrote this error message! "I should separate these out in a way that it's easy to add things and track changes on what we add" 🤣